### DISCUSSION TITLE
Using the tests

### DISCUSSION THREAD

**Discussion by nilsingwersen** [FriendsOfTYPO3/extension_builder#580] [Q&A] [answered]
How do I use the tests generated by the exentsion_builder? I tried setting up the tests like it is described in the testing documentation for typo3. That didn't work for me. I'm using TYPO3 11 in ddev on windows. Are there any extension where I can look at how it is handled there?

**Comment by simonschaufi** [comment, marked as answer]
You can have a look at the extension builder itself for example. I have added 2 composer scripts in the composer.json and I also run them in a github action.

**Reply by nilsingwersen** [reply]
That worked. Thank you. 😀
